### ISSUE TITLE
[IntelliJ][ReportedByUser] ProxySelector.setDefault() was changed to com.btr.proxy.selector.whitelist.ProxyBypassListSelector other than com.intellij.util.proxy.CommonProxy.ourInstance.

### ISSUE THREAD

**Issue by jpedersen-tech** [microsoft/azure-tools-for-java#6154]
IntelliJ build version: 2021.2.3 IU-212.5457.46
OS: Mac OS X
JDK: JetBrains s.r.o. 11.0.12
Plugin version: 3.58.0-2021.2
Additional Info: None
Parent component: 
```
com.intellij.openapi.wm.impl.IdeRootPane[,0,0,1280x775,layout=javax.swing.JRootPane$RootLayout,alignmentX=0.0,alignmentY=0.0,border=com.intellij.util.ui.UIUtil$1@1789fb51,flags=449,maximumSize=,minimumSize=,preferredSize=]
```
Error message: 
```
IdeaLoggingEvent[message=null, throwable=java.lang.Throwable: ProxySelector.setDefault() was changed to [com.btr.proxy.selector.whitelist.ProxyBypassListSelector@795f029c] - other than com.intellij.util.proxy.CommonProxy.ourInstance.
This will make some IDEA network calls fail.
	at com.microsoft.applicationinsights.core.dependencies.http.impl.conn.SystemDefaultRoutePlanner.determineProxy(SystemDefaultRoutePlanner.java:94)
	at com.microsoft.applicationinsights.core.dependencies.http.impl.conn.DefaultRoutePlanner.determineRoute(DefaultRoutePlanner.java:78)
	at com.microsoft.applicationinsights.core.dependencies.http.impl.client.InternalHttpClient.determineRoute(InternalHttpClient.java:125)
	at com.microsoft.applicationinsights.core.dependencies.http.impl.client.InternalHttpClient.doExecute(InternalHt

**Comment by Flanker32** [comment]
Uncaught Exception  java.lang.Throwable: ProxySelector.setDefault() was changed to [com.btr.proxy.selector.whitelist.ProxyBypassListSelector@795f029c] - other than com.intellij.util.proxy.CommonProxy.ourInstance.

**Comment by wangmingliang-ms** [comment]
I cannot reproduce this issue in the latest versions.
this issue should have been resolved in versions after 3.65.2, we didn't see same errors in later versions according to telemetry data.

**Comment by wangmingliang-ms** [comment]
close as the bug is not reproduceable and this issue is no longer active, feel free to reopen it when you encounter it again.
